code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -382,6 +382,41 @@
 	icon_open = "varsity_brown_open"
 	icon_closed = "varsity_brown"
 
+/obj/item/clothing/suit/storage/toggle/varsity/med
+	name = "medical varsity"
+	desc = "A varsity jacket in blue, red, and white, featuring an image of the Caduceus on the back, symbolizing not only the art of healing but also your fraternity (or sorority)"
+	icon_state = "varsity_med"
+	icon_open = "varsity_med_open"
+	icon_closed = "varsity_med"
+
+/obj/item/clothing/suit/storage/toggle/varsity/cargo
+	name = "cargo varsity"
+	desc = "An old, worn but loyal and reliable varsity jacket in faded white and yellow. Comes with extra pockets for convenience!"
```

Comment:
>Comes with extra pockets

Does it really? The description could be confusing in terms of the actual storage mechanics.

Reply:
It does visually, but not mechanically. I totally forgot about actual pocket mechanic and can see why this could be confusing. I guess I will either delete this part or change to "Comes with extra pockets for visual display only"